### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by stefanionescu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,100 +1,81 @@
...
-
-`auction-keeper` can participate in `flip` (collateral sale), `flap` (MKR buy-and-burn)
-and `flop` (MKR minting) auctions. Its unique feature is the ability to plug in external
-_bidding models_, which tell the keeper when and how high to bid. This keeper can be safely
-left running in background. The moment it notices or starts a new auction it will spawn a new instance
-of a _bidding model_ for it and then act according to its instructions. _Bidding models_ will
-be automatically terminated by the keeper the moment the auction expires.  The keeper also
-automatically `deal`s expired auctions if it's us who won them.
+`auction-keeper` can participate in collateral, surplus and debt auctions. Its unique feature is the ability to plug in external
+_bidding models_, which tell the keeper when and how high to bid. This keeper can be safely left running in background. The moment it nobidExpiryes or starts a new auction it will spawn a new instance of a _bidding model_ for it and then act according to its instructions. _Bidding models_ will
```
Comment (bkellerman) [inline README.md:9]: errant substitution

File: README.md [hunk 2]
```diff
@@ -1,100 +1,81 @@
...
+be automabidExpiryally terminated by the keeper the moment the auction expires.  The keeper also automabidExpiryally settles expired auctions if it's us who won them.
```
Comment (bkellerman) [inline README.md:10]: errant substitution

File: README.md [hunk 3]
```diff
@@ -1,100 +1,81 @@
...
-
 ### Monitoring ongoing auctions and discovering new ones
 
 The main task of this keeper, as already outlined above, is to constantly monitor all ongoing auctions,
 discover new ones, ensure that an instance of _bidding model_ is running for each auction, provide
 these instances of the current status of their auctions and bid according to decisions taken by them.
 
-The way the auction discovery and monitoring mechanism works at the moment is simplistic for illustration purposes.
-It basically operates as a loop which kicks in on every new block enumerating all auctions from `1` to `kicks`.
+The way the auction discovery and monitoring mechanism works at the moment is simplisbidExpiry for illustration purposes.
```
Comment (bkellerman) [inline README.md:32]: errant substitution

File: README.md [hunk 4]
```diff
@@ -1,100 +1,81 @@
...
+It basically operates as a loop which kicks in on every new block enumerating all auctions from `1` to `auctionsStarted`.
 Bidding models are checked every 2 seconds and submitted where appropriate.
 
-
 ### Starting and stopping _bidding models_
 
 `auction-keeper` maintains a collection of child processes, as each _bidding model_ is its own dedicated
 process. New processes (new _bidding model_ instances) are spawned by executing a command according to the
-`--model` commandline parameter. These processes are automatically terminated (via `SIGKILL`) by the keeper
+`--model` commandline parameter. These processes are automabidExpiryally terminated (via `SIGKILL`) by the keeper
```
Comment (bkellerman) [inline README.md:40]: errant substitution

File: README.md [hunk 5]
```diff
@@ -1,100 +1,81 @@
...
 shortly after their associated auction expires.
 
-Whenever the _bidding model_ process dies, it gets automatically respawned by the keeper.
+Whenever the _bidding model_ process dies, it gets automabidExpiryally respawned by the keeper.
```
Comment (bkellerman) [inline README.md:43]: errant substitution

File: README.md [hunk 6]
```diff
@@ -339,13 +295,12 @@ You can then run all tests with:
...
 
 ### Disclaimer
 
 YOU (MEANING ANY INDIVIDUAL OR ENTITY ACCESSING, USING OR BOTH THE SOFTWARE INCLUDED IN THIS GITHUB REPOSITORY) EXPRESSLY UNDERSTAND AND AGREE THAT YOUR USE OF THE SOFTWARE IS AT YOUR SOLE RISK.
-THE SOFTWARE IN THIS GITHUB REPOSITORY IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+THE SOFTWARE IN THIS GITHUB REPOSITORY IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARbidExpiryULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
```
Comment (bkellerman) [inline README.md:305]: errant substitution

**Review by bkellerman — CHANGES_REQUESTED** [review]
Just a few inadvertent subsititution.

Other than that, lgtm.

**Comment by stefanionescu** [comment]
Should be fixed

**Pull request merged**